invenio_access/utils.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Utility functions for Invenio-Access."""
+
+from __future__ import absolute_import, print_function
+
+from flask_principal import Identity, RoleNeed, UserNeed
+
+
+def get_identity(user):
+    """Create an identity for a given user instance."""
+    identity = Identity(user.id)
+
+    if hasattr(user, 'id'):
```

Comment:
Hehe, no :-) It's because of the [corresponding function](https://github.com/mattupstate/flask-security/blob/develop/flask_security/core.py#L252-L259) in Flask-Security. Actually another issue is that all the other ``on_identity_changed`` functions are not being run, but I didn't find any nice way of doing that. 
I created the function because it complicates the usage documentation a lot, if you don't have an easy way to get an identity like you would have during request handling.